ext/AvizExt/theme.jl

```diff
@@ -82,4 +85,118 @@ function scenario_colors!(obs_color::Observable, color_map::Vector, scen_types::
 
     color_map[hide] .= ((:white, 0.0),)
     obs_color[] = color_map
+
+"""
+    cluster_colors(clusters::Vector{Int64}, unique_colors::Vector{RGBA{Float32}})::Vector{RGBA{Float32}}
+    cluster_colors(clusters::Vector{Int64})::Vector{RGBA{Float32}}
+    cluster_colors(clusters::BitVector)::Vector{RGBA{Float32}}
+
+Gets colors for each clustered scenario.
+
+# Arguments
+- `clusters` : Vector with scenario cluster numbers
+
+# Returns
+Vector with one color for each clustered scenario.
+"""
+function cluster_colors(
+    clusters::Vector{Int64}, unique_colors::Vector{RGBA{Float32}}
+)::Vector{RGBA{Float32}}
+    unique_clusters::Vector{Int64} = sort(unique(clusters))
+    colors_map::Dict{Int64,RGBA{Float32}} = Dict(
+        c => unique_colors[i] for (i, c) in enumerate(unique_clusters)
+    )
+
+    colors::Vector{RGBA{Float32}} = Vector{RGBA{Float32}}(undef, length(clusters))
+    for (idx_c, cluster) in enumerate(clusters)
+        colors[idx_c] = colors_map[cluster]
+    end
+
+    return colors
+end
+function cluster_colors(clusters::Vector{Int64})::Vector{RGBA{Float32}}
+    unique_colors::Vector{RGBA{Float32}} = categorical_colors(
+        :seaborn_bright, length(unique(clusters))
+    )
+    return cluster_colors(Int64.(clusters), unique_colors)
+end
+function cluster_colors(clusters::BitVector)::Vector{RGBA{Float32}}
+    if unique(clusters) == [0]
+        return fill(parse(Colorant, :red), length(clusters))
+    elseif unique(clusters) == [1]
+        return fill(parse.(Colorant, :blue), length(clusters))
+    end
+    unique_colors::Vector{RGBA{Float32}} = parse.(Colorant, [:red, :blue])
+    return cluster_colors(Int64.(clusters), unique_colors)
+end
+
+"""
+    cluster_alphas(clusters::Vector{Int64})::Vector{Float64}
+    cluster_alphas(clusters::BitVector)::Vector{Float64}
+
+Get color alphas for each cluster weighted by number of scenarios.
+
+# Arguments
+- `clusters` : Vector with scenario cluster numbers
```

Comment:
```suggestion
- `clusters` : Vector with scenario cluster ids
```

Reply:
I think `id` is also ambiguous - that's why I avoided it in first place. It could be misinterpreted by the vector ID. What do you think of `tag`, `label` or `key`?

Reply:
`label` can also be ambiguous because of the axis labels.

Reply:
That said, I think I like `cluster_labels` more